Source/Core/VideoCommon/AbstractTexture.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright 2009 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <algorithm>
+
+#include "VideoCommon/AbstractTexture.h"
+
+AbstractTexture::AbstractTexture(const TextureConfig& c) : config(c)
+{
+}
+
+AbstractTexture::~AbstractTexture()
```

Comment:
You're saying AbsstractTexture::~AbstractTexture() = default can go in the cpp?  I never knew that!